### PR TITLE
Update openvpn container to v2.4.8

### PR DESCRIPTION
**Pull request by youssefazrak**
**What this PR does / why we need it**:
Openvpn update to v2.4.8. Security fixes, improvements etc.

This will update:
_ Openvpn for the user-clusters (control plane side and on worker nodes)
_ vpnsidecar
_ openshift addons

**Documentation**:

Release note: 
- https://github.com/OpenVPN/openvpn/releases/tag/v2.4.8
- https://github.com/OpenVPN/openvpn/releases/tag/v2.4.7

**Does this PR introduce a user-facing change?**:

```release-note
Update openvpn to 2.4.8
```

### CODE REVIEW

File: containers/openvpn/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM alpine:3.9
```
Comment (irozzo-1A) [inline containers/openvpn/Dockerfile:1]: Why not setting the package versions explicitly, especially openvpn?

File: containers/openvpn/release.sh [hunk 2]
```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-ver=v0.5
```
Comment (irozzo-1A) [inline containers/openvpn/release.sh:3]: would't be worthy to use the same versioning of openvpn?

File: containers/openvpn/Dockerfile (hunk 1 above)
Comment (youssefazrak) [reply containers/openvpn/Dockerfile:1]: Alpine 3.11 comes by default with Openvpn 2.4.8.

File: containers/openvpn/release.sh (hunk 2 above)
Comment (youssefazrak) [reply containers/openvpn/release.sh:3]: That seems logical. @xrstf what do you think?

File: containers/openvpn/Dockerfile (hunk 1 above)
Comment (irozzo-1A) [reply containers/openvpn/Dockerfile:1]: Be aware that `3.9` is a floating tag and that in general when you use a dependency manager (e.g. `apk`) you have no guarantee that the version you get today is the same you'll get tomorrow, unless you specify explicitly. I think that being explicit about the version makes the creation of the image more determinist and reproducible, and this is important in particular if we decide to follow the same versioning of openvpn.

Comment (youssefazrak) [reply containers/openvpn/Dockerfile:1]: Fair point. Package version is now specified for Openvpn.

File: containers/openvpn/release.sh (hunk 2 above)
Comment (youssefazrak) [reply containers/openvpn/release.sh:3]: Updated to 2.4.8

**Pull request merged**